src/components/app/components/Navbar.js

```diff
@@ -71,5 +78,12 @@ export default class Navbar extends React.Component {
 }
 
 Navbar.propTypes = {
+    actions: PropTypes.object,
     links: PropTypes.arrayOf(PropTypes.node),
 };
+
+const mapDispatchToProps = dispatch => ({
+    actions: bindActionCreators({ removeCookie }, dispatch),
+});
+
+export default connect(() => ({}), mapDispatchToProps)(Navbar);
```

Comment:
This is more neat and accepted  `export default connect(null, mapDispatchToProps)(Navbar);`

Reply:
Alright